src/pages/versions/config.json

```diff
@@ -50,10 +50,10 @@
       "nextLink": "/docs/intro/about",
       "latestData": [
         {
-          "versionLabel": "2.1.0",
-          "docUrl": "/docs/2.1.0/introduction",
-          "downloadUrl": "https://github.com/apache/incubator-seatunnel/releases/tag/2.1.0",
-          "sourceTag": "2.1.0"
+          "versionLabel": "2.1.1",
+          "docUrl": "/docs/2.1.1/introduction",
```

Comment:
Change `"docUrl": "/docs/2.1.1/introduction",` to `"docUrl": "/docs/2.1.1/intro/about",`